src/tree/updater_quantile_hist.h

```diff
@@ -365,11 +382,23 @@ class QuantileHistMaker: public TreeUpdater {
     DataLayout data_layout_;
 
     common::Monitor builder_monitor_;
-    common::ParallelGHistBuilder hist_buffer_;
-    rabit::Reducer<GradStats, GradStats::Reduce> histred_;
+    common::ParallelGHistBuilder<GradientSumT> hist_buffer_;
+    rabit::Reducer<xgboost::detail::GradientPairInternal<GradientSumT>,
```

Comment:
For ```xgboost::detail::GradientPairInternal<GradientSumT>``` - better to create additional ```using``` as above.

Reply:
fixed. thanks